plugins/module_utils/linode_common_info.py

```diff
@@ -220,10 +231,14 @@ def run(self) -> None:
         """
         Initializes and runs the info module.
         """
-        attribute_names = [v.name for v in self.attributes]
+        attribute_names = (
+            [[v.name for v in self.attributes]]
+            if len(self.attributes) > 0
+            else None
+        )
 
         super().__init__(
             module_arg_spec=self.module_arg_spec,
-            required_one_of=[attribute_names],
-            mutually_exclusive=[attribute_names],
+            required_one_of=attribute_names,
+            mutually_exclusive=attribute_names,
```

Comment:
The previous implementation brought an edge case that when the module didn't require any attribute, we still send an `[[]]` value as `required_one_of`. It caused a bug to fail the module.